### PR TITLE
Merge sdf14 to main

### PR DESCRIPTION
**Pull request by scpeters**
# ➡️ Forward port

Port sdf14 to main

Branch comparison: https://github.com/gazebosim/sdformat/compare/main...sdf14

**Note to maintainers**: Remember to **Merge** with commit (not squash-merge or rebase)

### CODE REVIEW

**Comment by scpeters** [comment]
jammy CI is unhappy as the `gz sdf` tests have extra console output `Library error: libgz-tools2-backward.so.2.0.0 not found. Improved backtrace generation will be disabled`, causing the expected string comparisons to fail

**Comment by scpeters** [comment]
> jammy CI is unhappy as the `gz sdf` tests have extra console output `Library error: libgz-tools2-backward.so.2.0.0 not found. Improved backtrace generation will be disabled`, causing the expected string comparisons to fail

I believe this was fixed by https://github.com/gazebo-release/gz-tools2-release/pull/15; rerunning failed CI

File: README.md [hunk 1]
```diff
@@ -10,16 +10,16 @@ Note: The branch name in the codecov URL & library version should be updated whe
 -->
 Build | Status
 -- | --
-Test coverage | [![codecov](https://codecov.io/gh/gazebosim/sdformat/branch/main/graph/badge.svg)](https://codecov.io/gh/gazebosim/sdformat/branch/main)
-Ubuntu Focal | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=sdformat-ci-main-focal-amd64)](https://build.osrfoundation.org/job/sdformat-ci-main-focal-amd64)
-Homebrew      | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=sdformat-ci-main-homebrew-amd64)](https://build.osrfoundation.org/job/sdformat-ci-main-homebrew-amd64)
-Windows       | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=sdformat-ci-main-windows7-amd64)](https://build.osrfoundation.org/job/sdformat-ci-main-windows7-amd64)
+Test coverage | [![codecov](https://codecov.io/gh/gazebosim/sdformat/tree/sdf14/graph/badge.svg)](https://codecov.io/gh/gazebosim/sdformat/tree/sdf14)
```
Comment (azeey) [inline README.md:13]: Revert to `main`.

**Review by azeey — APPROVED** [review]
Just one minor comment

**Pull request merged**